src/amber/environment/logger.cr

```diff
@@ -0,0 +1,10 @@
+require "logger"
+require "colorize"
+
+module Amber::Environment
+  class Logger < ::Logger
+    def puts(message, program = "Amber", color = :magenta)
+      log(self.level, message, "#{program} |".colorize(color))
```

Comment:
@eliasjpr Can we be able to disable colored output? maybe using `--no-color` flag already implemented on #272

Reply:
I will take a look at this, but I a mostly certain this can be done